### DISCUSSION TITLE
How to disable .paddleocr directory creation?

### DISCUSSION THREAD

**Discussion by Ocean-Moist** [PaddlePaddle/PaddleOCR#12774] [Q&A]
### 请提出你的问题 Please ask your question

In PaddlePaddle/Paddle#48873 they say to disable .paddleocr file creation to get paddleocr to work in the docker container, but I can not find anything in the docs about how to do this. I can also do ENV HOME=/tmp but I do not want to spend time downloading files unnecessarily.

**Comment by UserWangZz** [comment]
When you do not specify the default detection and recognition model address, paddleocr will download the official model by default. You can delete it manually, or it will not be downloaded the second time after downloading.

**Comment by Ocean-Moist** [comment]
To avoid re-downloading these files each time a new container is created, is it possible to pre-download the files and store them in a local directory? This way, when a new container is started, it can copy the files from the local directory instead of downloading them again.

**Reply by SWHL** [reply]
It could be, but need to wait the new version.

**Reply by ajayghiyad** [reply]
yes, just download the tar files
and ensure that you extract stuff in relevant folders 
attaching sample docker for en_PP-OCR model

```
# Create directories
RUN mkdir -p RUN mkdir -p /root/.paddleocr/whl/rec/en/en_PP-OCRv3_rec_infer

# Copy the tar files
COPY en_PP-OCRv3_rec_infer.tar /root/.paddleocr/whl/rec/en/en_PP-OCRv3_rec_infer/en_PP-OCRv3_rec_infer.tar

# Extract the contents of the .tar files
RUN tar -xf /root/.paddleocr/whl/rec/en/en_PP-OCRv3_rec_infer/en_PP-OCRv3_rec_infer.tar -C /root/.paddleocr/whl/rec/en/en_PP-OCRv3_rec_infer/
```

**Comment by saikumar305** [comment]
Anyone has found solution for this?
